File: CinemaRest.Service/CinemaRest.Service/Models/Seat.cs
using System;

namespace CinemaRest.Service.Models
{
    public class Seat : LinkResourceBase
    {
        public Guid Id { get; set; } = Guid.NewGuid();
        public Screen Screen { get; set; }
        public int Row { get; set; }
        public int SeatNumber { get; set; }

        public Seat() { }
    }
}

File: CinemaRest.Client/src/cinemarest/client/models/Seat.java
package cinemarest.client.models;

public class Seat {
    private String id;
    private int row;
    private int seatNumber;
    private Screen screen;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public int getRow() {
        return row;
    }

    public void setRow(int row) {
        this.row = row;
    }

    public int getSeatNumber() {
        return seatNumber;
    }

    public void setSeatNumber(int seatNumber) {
        this.seatNumber = seatNumber;
    }

    public Screen getScreen() {
        return screen;
    }

    public void setScreen(Screen screen) {
        this.screen = screen;
    }
}
